### PR TITLE
update explorer 3.4.0 release notes

### PR DESCRIPTION
**Pull request by cooper-lzy**
### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Review by whitewum — APPROVED** [review]

**Review by abby-cyber — APPROVED** [review]

File: docs-2.0/20.appendix/release-notes/explorer-release-note.md [hunk 1]
```diff
@@ -1,38 +1,47 @@
...
-  - Support [custom icon library and edge style](../../nebula-explorer/canvas-operations/canvas-overview.md). Explorer has ten built-in icons for industry categories and supports uploading pictures as an icon.
-  - Support [nGQL template](../../nebula-explorer/db-management/ngql-template.md). Design the query statement template directly or make nGQL statement as a template in the console, and then directly call the template and fill in the parameters to query.
-  - Support [database user management](../../nebula-explorer/db-management/dbuser_management.md). Manage database users visually, including creating and authorizing users.
-  - Workflow added [node2vec algorithm](../../graph-computing/algorithm-description.md).
-  - (Beta) Support [OAuth 2.0 authentication](../../nebula-explorer/deploy-connect/ex-ug-connect.md). Users can log in to Explorer only after being authenticated by OAuth2.0 to ensure data security.
+
+  - Support viewing the [creation statements](../../nebula-explorer/db-management/10.create-schema.md) of the schema.
+  - Add beta function switch on the global settings page.
```
Comment (abby-cyber) [inline docs-2.0/20.appendix/release-notes/explorer-release-note.md:8]: Add a **Beta functions** button on the global settings page.

File: docs-2.0/20.appendix/release-notes/explorer-release-note.md [hunk 2]
```diff
@@ -1,38 +1,47 @@
...
+  - Add product feedback page.
```
Comment (abby-cyber) [inline docs-2.0/20.appendix/release-notes/explorer-release-note.md:9]: Add a product feedback page.

File: docs-2.0/20.appendix/release-notes/explorer-release-note.md [hunk 3]
```diff
@@ -1,38 +1,47 @@
...
-  - Optimize page loading speed.
+
+  - Remove the timeout limit for slow queries.
+  - Keep history on the console page after switching pages.
+  - Support adding comments with `#` on the console page.
+  - Support adding comments with `#` or `//` when creating nGQL templates.
+  - Update the global settings page.
+  - Support the visual modification of the IP whitelist.
+  - Show VID on canvas by default.
+  - Browser compatibility hint.
```
Comment (abby-cyber) [inline docs-2.0/20.appendix/release-notes/explorer-release-note.md:20]: Display browser compatibility hints.

File: docs-2.0/20.appendix/release-notes/explorer-release-note.md [hunk 4]
```diff
@@ -1,38 +1,47 @@
...
+  - Show the kernel version in the connection information.
+  - Add indexes to the built-in dataset.
+  - Optimize the login page.
+  - Optimize Workflow:
+    - Add algorithm descriptions.
+    - Optimize the parameter configurations of the graph algorithm.
+    - Optimize the presentation of the result.
+  - Optimize interaction:
```
Comment (abby-cyber) [inline docs-2.0/20.appendix/release-notes/explorer-release-note.md:28]: interactions

File: docs-2.0/20.appendix/release-notes/explorer-release-note.md [hunk 5]
```diff
@@ -1,38 +1,47 @@
...
+    - Vertex filter
+    - Query by tag
+    - Search path
+  - Optimize presentation:
```
Comment (abby-cyber) [inline docs-2.0/20.appendix/release-notes/explorer-release-note.md:32]: presentations

File: docs-2.0/20.appendix/release-notes/explorer-release-note.md [hunk 6]
```diff
@@ -1,38 +1,47 @@
...
+    - Optimize the presentation of schema statistics.
+    - Optimize the layout of force.
+    - Optimize the layout of the visual query results after importing them to the canvas.
+    - Optimize the presentation of vertices on dangling edges.
+    - Optimize the console page.
+  - Optimize hint:
```
Comment (abby-cyber) [inline docs-2.0/20.appendix/release-notes/explorer-release-note.md:38]: hints

**Pull request merged**